src/Bugsnag/Payload/Device.cs

```diff
@@ -44,5 +46,23 @@ private static string OsName
 #endif
       }
     }
+
+    private static Dictionary<string, string> RuntimeVersions
+    {
+      get
+      {
+        var runtimeVersions = new Dictionary<string, string>();
+#if NET45
+        // The framework description is only available from .NET 4.5 onwards and will not return
+        // an accurate value for .NET Core until version 3.0 
+        runtimeVersions.AddToPayload("dotnet", RuntimeInformation.FrameworkDescription);
```

Comment:
The [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription) for `RuntimeInformation.FrameworkDescription` says the minimums are:

* .NET Core: 1.0
* .NET Framework: 4.7.1
* .NET Standard: 1.1

Should we not use these preprocessor directives to condition each one?
```
#if NETCOREAPP1_0 || NET471 || NETSTANDARD1_1
```

Reply:
I think we should only include it for the .NET Framework for now because this won't show an accurate version for the .NET Core/.NET Standard [until version 3.0 is released](https://github.com/dotnet/coreclr/issues/22844). 

As for .NET 4.5 vs 4.7.1, the code compiles ok for 4.5 so it might be worth seeing what is actually recorded when running with that version.